### PR TITLE
BAU: Import configurations for policy, config and saml-soap-proxy

### PR DESCRIPTION
**Pull request by rhowe-gds**
These services have generic configurations which do not differ between
environments, save for values sourced from environment variables. Move them
into this repository and the service packages.

### CODE REVIEW

File: debian/saml-soap-proxy/saml-soap-proxy.yml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      - type: logstash-file
+        currentLogFilename: /var/log/ida/saml-soap-proxy_access.log
+        archivedLogFilenamePattern: /var/log/ida/saml-soap-proxy_access.log.%d.gz
+        archivedFileCount: 7
+      - type: logstash-syslog
+        facility: local2
+        tag: saml-soap-proxy
+
+saml:
+  entityId: https://signin.service.gov.uk
```
Comment (JHjava) [inline debian/saml-soap-proxy/saml-soap-proxy.yml]: Should the entityId and the eidasEntityId be environment variables?

Comment (willp-bl) [reply debian/saml-soap-proxy/saml-soap-proxy.yml]: Yes

File: debian/saml-soap-proxy/saml-soap-proxy.yml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+      archivedFileCount: 7
+    - type: logstash-syslog
+      facility: local2
+      tag: saml-soap-proxy
+    - type: raven
+      dsn: ${SENTRY_DSN:-naive+https://sentry/1}
+      threshold: ERROR
+      tags: service-name:saml-soap-proxy
+
+metadata:
```
Comment (JHjava) [inline debian/saml-soap-proxy/saml-soap-proxy.yml]: The signin.service of the uri and the expectedEntityId should also be environment variables.

File: debian/saml-soap-proxy/saml-soap-proxy.yml (hunk 1 above)
Comment (rhowe-gds) [reply debian/saml-soap-proxy/saml-soap-proxy.yml]: Yes, I was clearly not paying enough attention when putting this PR together. This is an old version of the configuration. I've redone the PR now with the correct config file.

Good spot @JHjava

File: debian/saml-soap-proxy/saml-soap-proxy.yml (hunk 2 above)
Comment (rhowe-gds) [reply debian/saml-soap-proxy/saml-soap-proxy.yml]: Fixed. I'd brought in a stale version of the config - whoops! Top spot!

**Comment by willp-bl** [comment]
Can it be enabled for all environments now? Stub-country already exists everywhere

**Comment by rhowe-gds** [comment]
Yes, something like that. Currently the only differences in those apps' config are the eIDAS configuration section. If that difference can be removed, we can move those configurations into this repo as well.

**Comment by rhowe-gds** [comment]
I know that difference can't be removed currently, but if we can find a way to make it not be different (be it an enable flag or shipping eidas config everywhere or something else) then we can move those configurations into this repository. I wasn't saying to just delete the differences - obviously that doesn't fly

**Pull request merged**